clang/test/CodeGen/arithmetic-fence-builtin.c

```diff
@@ -1,20 +1,30 @@
 // Test with fast math
 // RUN: %clang_cc1 -triple i386-pc-linux-gnu -emit-llvm -DFAST \
 // RUN: -mreassociate \
-// RUN: -o - %s | FileCheck --check-prefixes CHECK,CHECKFAST,CHECKNP %s
+// RUN: -o - %s | FileCheck --check-prefixes CHECK,CHECKFAST,CHECKPREC,CHECKNP %s
```

Comment:
What does `CHECKPREC` mean?

Reply:
CHECKPREC (for check precision) used in the #pragma float_control(precise on).